web/src/resources/MembersResource.ts

```diff
@@ -23,9 +22,46 @@ export async function fetchAllMembers(orgnames: string[], signal: AbortSignal) {
 export async function fetchMembersForOrg(
   orgname: string,
   signal: AbortSignal,
-): Promise<IMember[]> {
+): Promise<IMembers[]> {
   const getMembersUrl = `/api/v1/organization/${orgname}/members`;
   const response = await axios.get(getMembersUrl, {signal});
   assertHttpCode(response.status, 200);
   return response.data?.members;
 }
+
+export async function fetchCollaboratorsForOrg(
+  orgname: string,
+  signal: AbortSignal,
+): Promise<IMembers[]> {
+  const getCollaboratorsUrl = `/api/v1/organization/${orgname}/collaborators`;
+  const response = await axios.get(getCollaboratorsUrl, {signal});
+  assertHttpCode(response.status, 200);
+  return response.data?.collaborators;
+}
+
+export async function fetchTeamMembersForOrg(
+  org: string,
+  teamName: string,
+  signal?: AbortSignal,
+) {
+  const teamMemberUrl = `/api/v1/organization/${org}/team/${teamName}/members?includePending=true`;
+  const teamMembersResponse = await axios.get(teamMemberUrl, {signal});
+  assertHttpCode(teamMembersResponse.status, 200);
+  return teamMembersResponse.data?.members;
+}
+
+export async function deleteTeamMemberForOrg(orgName: string, teamName: string, memberName: string) {
+  try {
+    await axios.delete(`/api/v1/organization/${orgName}/team/${teamName}/members/${memberName}`);
+  } catch (err) {
+    console.error(`Unable to delete member for team: ${teamName}`, err);
```

Comment:
This swallows the error if something goes wrong with the request. React-query won't be able to catch this.

Reply:
replaced with:
```
catch(error){
    throw new ResourceError('Unable to delete team', teamName, error);
  }
  ```